web/themes/joinup/joinup_theme.theme

```diff
@@ -257,10 +257,6 @@ function joinup_theme_theme($existing, $type, $theme, $path) {
       'render element' => 'form',
       'path' => $path . '/templates/form',
     ],
-    'join_collection_form' => [
-      'render element' => 'form',
-      'path' => $path . '/templates/form',
```

Comment:
I had to remove this form override because it was preventing the form to be correctly rebuilt in the AJAX callback. This took me a lot of time to find. I have replaced this with a custom `collection_theme_button` theme function which works fine in AJAX.